src/output.rs

```diff
@@ -50,14 +50,14 @@ pub fn get_target_output_writable(output_file: Option<&Path>) -> Box<dyn Write>
                 if let Some(parent_path) = parent {
                     if !parent_path.exists() {
                         panic!(
-                            "Cannot create {} to dump coverage data, as {} doesn't exist",
+                            "Cannot create file {} to dump coverage data, as the parent directory {} doesn't exist.",
                             output.display(),
                             parent_path.display()
                         )
                     }
                 }
                 panic!(
-                    "Cannot create the file {} to dump coverage data.",
```

Comment:
Could you revert the change on this line?

Reply:
I already reverted that change. Line 60 in PR matches with [line 60 in the master branch](https://github.com/mozilla/grcov/blob/725054c044a59138e4670f10cf221d9d310e226d/src/output.rs#L60).

Reply:
There is a diff here, it looks like you removed "the "

Reply:
Sorry about that, I've added "the".